components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/ActionExecutorConfig.java

```diff
@@ -154,6 +154,10 @@ private List<String> getPropertyValues(String propertyKey) {
             return Collections.emptyList();
         }
 
+        if (propertyValue instanceof String) {
+            return Collections.singletonList(propertyValue.toString());
+        }
+
```

Comment:
Is there reason why should be allow to configure single value as String instead of list of one element.

Reply:
The API contract is to get a list of property values. So we are returning a list adhering to that.